### PR TITLE
OT-3429 Precalcless tracking and refine not available if precalc exists

### PR DESCRIPTION
**Pull request by amilovsky**

### CODE REVIEW

File: keentools/geotracker/camera_input.py [hunk 1]
```diff
@@ -114,13 +116,17 @@ def first_frame(self) -> int:
         geotracker = get_current_geotracker_item()
         if not geotracker:
             return 1
-        return geotracker.precalc_start
+        if not geotracker.precalcless:
+            return geotracker.precalc_start
```
Comment (SergeyKrivohatskiy) [inline keentools/geotracker/camera_input.py:120]: Remove this. If precalc is used, precalc range is taken from file, not from image input

File: keentools/geotracker/camera_input.py [hunk 2]
```diff
@@ -114,13 +116,17 @@ def first_frame(self) -> int:
...
+        return bpy_start_frame()
 
     def last_frame(self) -> int:
         geotracker = get_current_geotracker_item()
         if not geotracker:
             return 0
-        return geotracker.precalc_end
+        if not geotracker.precalcless:
+            return geotracker.precalc_end
```
Comment (SergeyKrivohatskiy) [inline keentools/geotracker/camera_input.py:128]: same

**Review by SergeyKrivohatskiy — CHANGES_REQUESTED** [review]

File: keentools/geotracker/camera_input.py (hunk 1 above)
Comment (amilovsky) [reply keentools/geotracker/camera_input.py]: ok

File: keentools/geotracker/camera_input.py (hunk 2 above)
Comment (amilovsky) [reply keentools/geotracker/camera_input.py]: done

**Review by SergeyKrivohatskiy — APPROVED** [review]

**Pull request merged**